tester_app/config/initializers/assets.rb

```diff
@@ -9,3 +9,7 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+# Add client/assets/stylesheets to asset pipeline's search path.
+Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
+
```

Comment:
// If you do not want to move the images and fonts directories to be under /client/assets/, you could also consider 
// creating symlinks there that point to the original rails directories. In that case, you would not add these paths here.